modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Sequence.scala

```diff
@@ -204,6 +204,25 @@ object Sequence {
         case Final(st, _)  => Final(st, s)
       })
 
+    def isRunning(st: State): Boolean = SequenceState.isRunning(st.status)
```

Comment:
Added several utility functions here.

Reply:
Maybe these could be converted to `ops` via an implict class, but it's low prio